nemo_text_processing/inverse_text_normalization/fr/taggers/telephone.py

```diff
@@ -52,17 +52,36 @@ def __init__(self):
             (graph_ties + delete_hyphen + graph_digit),
         )
 
+        # accept `double zéro` -> `00`
+        single_digits = graph_digit | graph_zero
+        digit_words = pynini.union(graph_digit.optimize(), pynini.cross("zéro", "0")).invert()
```

Comment:
instead of hard coding, could you important from https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/inverse_text_normalization/fr/data/numbers/zero.tsv